ROADMAP.md

```diff
@@ -2,60 +2,47 @@
 
-The following table includes the current roadmap for Tanzu Community Edition. If you have any questions
-or would like to contribute, please attend a community meeting to discuss with our team. If you
-don't know where to start, we are always looking for contributors that will help us reduce
-technical, automation, and documentation debt. Please take the timelines and dates as proposals and goals.
-Priorities and requirements change based on community feedback, roadblocks encountered, community contributions,
-etc. If you depend on a specific item, we encourage you to attend community meetings to get updated status information,
-or help us deliver that feature by contributing to Tanzu Community Edition.
-
-Last Updated: October 2021
-
-|Theme|Feature|Timeframe|
-|--|--|--|
-|**Build user community**|Start public Office Hours and Community Meetings|0-3 months|
-||Start Slack channel and Google Group|0-3 months|
-||Training/enablement resources (including KubeAcademy and Tanzu Developer Center|0-3 months|
-||Reward community support|3-6 months|
-||Community roadmap prioritizatiion|3-6 months|
-|**Accelerate time to value**|Include demo applications for more complete startup experience|3-6 months|
-||Work to reduce time to cluster availability on vSphere and public clouds|3-6 months|
-||Pre-integrated multi-package solutions|3-6 months|
-|**Robust Kubernetes platform**|Limited-internet (including airgapped) support|0-3 months|
-||Aligned to upstream Kubernetes releases|0-3 months|
-||Early availability of upstream Kubernetes releases|3-6 months|
-|**Rich package library**|Work with community to solicit and prioritize desired packages|0-3 months|
-||Pursue community and ISV package lifecycle and configuration integrations via Carvel and Tanzu Framework|3-6 months|
-|**Tanzu scenario support beyond Kubernetes**|Tanzu Application Platform OSS components|0-3 months|
-||Add developer focused UI experience|0-3 months|
-||Bitnami integration via kubeapps|0-3 months|
+The following table includes the current roadmap for Tanzu Community Edition. If you have any questions or would like to contribute, please attend a community meeting to discuss with our team. If you don't know where to start, we are always looking for contributors that will help us reduce technical, automation, and documentation debt. Please take the timelines and dates as proposals and goals. Priorities and requirements change based on community feedback, roadblocks encountered, community contributions, etc. If you depend on a specific item, we encourage you to attend community meetings to get updated status information, or help us deliver that feature by contributing to Tanzu Community Edition.
+
+Last Updated:  January 2022
+
+| Theme | Feature | Status (Phase) | Targeted Release |
+| ----- | ------- | -------------- | ---------------- |
+| Build user community | Reward community support | n/a | ongoing |
+| |Community roadmap prioritization | n/a | ongoing |
```

Comment:
This is a roadmap item? Isn't this what the roadmap is?

Reply:
Nigel's activities. (In this case, the activity is to hand out swag for attendance and contribution.) Marketing activities. Follow-up to conversations that draw people (obviously not too effectively so far) to the channels and meetings. At the program level, deliverables are not all code and docs.

Reply:
Said another way, it doesn't make sense for a user to read the roadmap to see what's coming up and think something like "great, I can't wait for the v0.12.0 release when we get a community". This just seems like a very odd thing to have on a roadmap. A personal or team goal planning, sure. But a roadmap not so much.

Reply:
Yes, I get your point. In the previous version, this was one of several items in this category, so "Reward participation" as an activity made more sense in context. It also included several dev advocacy and technical marketing activities aimed at building the community. The content plan has not yet been approved by the program - it has, I think, by marketing - so I was asked to drop those items. I could drop the whole category till those things are approved, but since the (potential) community is one of the things we talk about a lot when we present TCE to Tanzu customers, press, analysts, etc. the general category should probably remain - we can revise to add the other deliverables as soon as the plan is approved. But I will discuss it with People Above My Head and see what they think.

Reply:
Removed.